sci-electronics/gazebo/gazebo-2.2.1.ebuild

```diff
@@ -40,13 +40,14 @@ DEPEND="${RDEPEND}
 	doc? ( app-doc/doxygen app-text/ronn )
 "
 
-PATCHES=(
-	"${FILESDIR}/${P}-stdint.patch"
-)
-
-SRCDIR="$(ls ${WORKDIR} | grep osrf-gazebo | tail -n 1)"
+SRCDIR="osrf-gazebo-3b8a07b268a5"
```

Comment:
I fixed, the variables outside the functions are calculated at the start of the emerge
if you put that variables inside, there are calculated at the right time and in the right folder